src/python/module/smartredis/dataset.py

```diff
@@ -229,3 +229,13 @@ def get_tensor_names(self):
         :rtype: list
         """
         return self._data.get_tensor_names()
+
+    @exception_handler
+    def get_tensor_dims(self, name):
+        """Get the dimensions of a tensor in the DataSet
+
+        :return: a list of the tensor dimensions
+        :rtype: list
```

Comment:
incredibly minor doc request: can we change the return type to `list[int]`

Reply:
done, and also changed the rtype for the other lists to list[str]